bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -238,50 +238,62 @@
         # or greater.
         let(:dependency_files) { bundler_project_dependency_files("legacy_ruby") }
         its([:version]) { is_expected.to eq(Gem::Version.new("1.4.6")) }
+      end
```

Comment:
Review here is easier with https://github.com/dependabot/dependabot-core/pull/3809/files?diff=unified&w=1